web-components/src/components/inputs/LocationCountryField.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import { Field } from 'formik';
+import iso3166 from 'iso-3166-2';
```

Comment:
Should we lazy load this component now, since this new lib is a bit heavy and is used only in inputs?

The idea is just to replace `LocationCountryField.tsx` import by `const LocationCountryField = lazy(() => import('./LocationCountryField'));` where it's used and wrap it with `<Suspense><LocationCountryField /></Suspense>`.

Reply:
Thanks, good advice! I accidentally merged to close the PR even though I had previously seen the review comments. They come in another little review PR

Reply:
Review PR: https://github.com/regen-network/regen-web/pull/1163